---
output:
  html_document:
    keep_md: TRUE
---

# Abstract  

Single-cell technologies are offering unprecedented insight into complex biology, revealing the behavior of rare cell populations that are typically masked in bulk population analyses. The application of these methodologies to cell fate reprogramming holds particular promise as the manipulation of cell identity is typically  inefficient,  generating  heterogeneous  cell  populations.  One  current  limitation  of  single-cell approaches is that lineage relationships are lost asa result of cell processing, restricting interpretations of  the  data  collected.  Here,  we  present  a single-cell  resolution clonal  tracking  approach,  based  on combinatorial  cell  indexing,  permitting  the  parallel  capture  of  lineage  information  and  cell  identity. "CellTagging" integrates with high-throughput single-cell RNA-sequencing, where multiple rounds of cell labeling  enable  the  construction  of  multi-level  lineage  trees.  We  apply this  technology  to  revealthe transcriptional dynamics of direct reprogramming from fibroblasts to induced endoderm progenitors. These  analyses  demonstrate  that  while  many  cells  initiate  reprogramming,  complete  silencing  of fibroblast  identity  and  transition  to  a  progenitor-like state  represents  a  rare  event.  Clonal  analyses uncover a remarkable degree of heterogeneity arising from individual cells. Overall, very few cells fully reprogram to generate expanded populations with a low degree of clonal diversity. Extended culture of these engineered cells reveals an instability of the reprogrammed state and reversion to a fibroblast-like phenotype.

[CellTag bioRxiv Preprint](https://www.biorxiv.org/content/early/2017/04/28/127860)  
[Morris Lab Website](http://morrislab.wustl.edu/)  
[Website inspired by DropViz](http://dropviz.org)
